qcengine/programs/turbomole/runner.py

```diff
@@ -0,0 +1,214 @@
+        commands = ["ridft"] if ri_calculation else ["dscf"]
+        # ricc2 will also calculate the gradient
+        if model.method in METHODS["ricc2"]:
+            commands.append("ricc2")
+        # Gradient calculation for DFT/HF
+        elif input_model.driver.derivative_int() == 1:
+            grad_command = "rdgrad" if ri_calculation else "grad"
+            commands.append(grad_command)
+        elif input_model.driver.derivative_int() == 2:
+            freq_command = "aoforce"
+            commands.append(freq_command)
+            # Add
+            #   $noproj
+            #   $nprhessian file=nprhessian
+            # to control file.
+            turbomolrec["outfiles"]["hessian"] = "nprhessian"
+
+        if input_model.driver.derivative_int() == 1:
+            turbomolrec["outfiles"]["gradient"] = "gradient"
+        command = ["; ".join(commands)]
+        turbomolrec['command'] = command
+
+        # TODO: check if the chosen commands are available with which()?
+
+        return turbomolrec
+
+    def execute(self,
+                inputs: Dict[str, Any],
+                *,
+                extra_outfiles=None,
+                extra_commands=None,
+                scratch_name=None,
+                timeout=None) -> Tuple[bool, Dict]:
+
+        success, dexe = execute(
+            inputs["command"],
+            inputs["infiles"],
+            inputs["outfiles"],
+            shell=True,
+            # TODO: scratch_messy?
+            # scratch_messy=False,
+        )
+        return success, dexe
+
+    def parse_output(self,
+                     outfiles: Dict[str, str],
+                     input_model: 'ResultInput') -> 'Result':  # lgtm: [py/similar-function]
+
+        stdout = outfiles.pop("stdout")
+
+        # nwmol, if it exists, is dinky, just a clue to geometry of nwchem results
+        qcvars, gradient, hessian = harvest(input_model.molecule, stdout, **outfiles)
+
+        if gradient is not None:
+            qcvars['CURRENT GRADIENT'] = gradient
+
+        if hessian is not None:
+            qcvars['CURRENT HESSIAN'] = hessian
+
+        retres = qcvars[f'CURRENT {input_model.driver.upper()}']
+        if isinstance(retres, Decimal):
+            retres = float(retres)
+        elif isinstance(retres, np.ndarray):
+            retres = retres.ravel().tolist()
+
+        output_data = {
+            'schema_name': 'qcschema_output',
+            'schema_version': 1,
+            'extras': {
+                'outfiles': outfiles,
+            },
+            'properties': {},
+            'provenance': Provenance(creator="Turbomole", version=self.get_version(),
+                                     routine="turbomole"),
+            'return_result': retres,
+            'stdout': stdout,
+        }
+
+        # got to even out who needs plump/flat/Decimal/float/ndarray/list
+        # Decimal --> str preserves precision
+        output_data['extras']['qcvars'] = {
```

Comment:
I don't think we need this anymore. We have serializations that can handle complex python types.

Double checking with @loriab.

Reply:
I commented this out for now in e851df6